### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by CuttlefishXuan** [CoinCheung/BiSeNet#120]
I am training the bisenetv2 model on cityscapes dataset, and I have trained it for 40k iterations. But the loss is about 6.5 and not decreased. lr_start = 5e-2, and lr=0.035 for now. Should I waiting for more iterations, is it normal?

**Comment by CoinCheung** [comment]
I think it is normal, since there are 4 auxiliary losses also computed during the training phase. Have you verified it?

**Comment by CuttlefishXuan** [comment]
Thanks, when the iteration=120k, the loss is about 5.7, and the eval mIOU is about 0.62.
Now my program is shutdown for some reason I don't know in the server. Maybe I have to train it from the begining. T_T

**Comment by CoinCheung** [comment]
I am closing this. You can still leave messages if you need more information.
